sycl/source/detail/graph_impl.cpp

```diff
@@ -907,8 +907,10 @@ exec_graph_impl::enqueue(const std::shared_ptr<sycl::detail::queue_impl> &Queue,
       // If we have no requirements or dependent events for the command buffer,
       // enqueue it directly
       if (CGData.MRequirements.empty() && CGData.MEvents.empty()) {
-        if (NewEvent != nullptr)
+        if (NewEvent != nullptr){
```

Comment:
Interesting that clang-format passed without picking this up
```suggestion
        if (NewEvent != nullptr) {
```